src/views/CardStack.js

```diff
@@ -571,6 +523,8 @@ class CardStack extends Component<DefaultProps, Props, void> {
     const SceneComponent = this.props.router.getComponentForRouteName(
       props.scene.route.routeName
     );
+    const extraProps = this.props.createExtraSceneProps &&
```

Comment:
This callback requires the `NavigationSceneRendererProps` of the current scene that is being rendered. The props is used to interpolate styles. Since `NavigationSceneRenderProps` is only available in the `render` callback of the Transitioner, I'm not sure how to move that to componentDidUpdate or componentWillUpdate?

What are the downsides of using it in the render path? Performance issue?